src/config_file.cc

```diff
@@ -188,7 +188,17 @@ void CONFIG_TREC::record_tag( string config_tag, string config_value ) {
 	found = 1;
 	}
   }
-  
+
+  if (config_tag == "lmparam_delta" ) {
+	lmparam_delta = strtod(config_value.c_str(), NULL);
```

Comment:
lm was indicating that this parameter belongs to Language Model. this parameter should be dirplus or just dirparam_delta and so on